packages/expo-system-ui/src/SystemUI.ts

```diff
@@ -28,3 +28,20 @@ export async function setBackgroundColorAsync(color: ColorValue): Promise<void>
 export async function getBackgroundColorAsync(): Promise<ColorValue | null> {
   return await ExpoSystemUI.getBackgroundColorAsync();
 }
+
+/**
+ * Restores the root view background color.
+ * Call this function in your apps root component to restore the most recently set background color.
+ *
+ * @example
+ * ```ts
+ * await SystemUI.restoreBackgroundColorAsync();
+ * ```
+ * @platform android
+ */
+export async function restoreBackgroundColorAsync(): Promise<void> {
```

Comment:
Yes, that'll work. I'll change that 👍 Do you think we need to add a function to clear the stored value? Reset it back to nil so it falls back to following the system theme.

Reply:
What do you think if we just let you do that with `setBackgroundColorAsync(null)`? The getter can already return `null` if it is unset, right? It would be consistent then 🤔

Reply:
Yes, the getter can return null so probably makes more sense to allow the setter to accept null. I'll document the behaviour when you do that.